search/retrieval-augmented-generation/examples/question_answering.ipynb

```diff
@@ -88,7 +91,7 @@
SZ+ZlKMJNcMxVg5i2Cta6vqV69etp+WVldw/TUMhVnprqqfv2bdODAAVW59ho9+OADJxmZxxbfffd9jf5ijD2v4a232FhNYpCKLBJZbu9PZzpeuPbTM42XzyGAAAIIIICAPwTokeWPdSJKBBAIkYDTCiqn4TitSPBiIsvpHDkPAa8I0CPLKytxLA6391NvzY5oEEAAAQQQQCBcAvTICpc810UAgagQcFqRQCIrKm4HJhlkARJZQQZmeAQQQAABBBBAwAMC9MjywCIQAgIIeEfAaQWV04idViSQyHIqynkIZC5AIstbd4fb+6m3Zkc0CCCAAAIIIBAuAXpkhUue6yKAgCcFnFZQuR08iSy3RRkvGgVIZHlr1cO1n3pLgWgQQAABBBBAwG0BemS5Lcp4CCDgawGnFVROJ+m0IoFEllNRzkMgcwESWd66O9zeT701O6JBAAEEEEAAgXAJ0CMrXPJcFwEEokLAaUUCiayouB2YZJAFSGQFGZjhEUAAAQQQQAABDwjQI8sDi0AICCDgHQGnFVROI3ZakXBxmbJPxCjmjXZt79ezzz7tdHjOQwCBdAL9+r2oIf8ZqoACT27etLE/OOEVcHs/De9suDoCCCCAAAIIeEWAHlleWQniQAABTwg4raByO9gyZcrdEZAmVr/uOo0YMdzt4RkPgagQiI+/TwkLFypGarRp04ZJUTFpD08yXPuph0kIDQEEEEAAAQRcEKBHlguIDIEAApEj4LSCyumMnVYkxMXFFUwJxO4x486fN0clS5ZwegnOQwABSVu3blPN62tbi9iYlEKJiYl7gQmvgNv7aXhnw9URQAABBBBAwCsC9MjyykoQBwIIRKTA6VQklC5ddoRiYlrweGFE3gpMKsgCqY8VKhAYmZS0MT7Il2N4BBBAAAEEEEAAgTAJ0CMrTPBcFgEEvCngtILKafSnU5EQFxd3bUogdokZ+6NBH6h+/ZucXobzEIhqgenTZ6hDx4esQWxMSpXExMSlUQ3ikcm7vZ96ZFqEgQACCCCAAAJhFqBHVpgXgMsjgIC3BE6ngioYkae+vTBHjhz64P13SWYFA5kxI0rAJLEe6tRZycnJUkB9kpI29I2oCfp4MuHeT31MR+gIIIAAAgggkIUAPbK4PRBAAIF0AqdTQeUE7kwqEkqXKT9ICnQw45vHDNu3b0fPLCfYnBNVAqYn1uDBQ+xbCo8dMR8lbVrfMaoQPD5Zt/dTj0+X8BBAAAEEEEAgRAL0yAoRNJdBAIHoFDjTioTUyqxUNfM2wwoVLlP+AvmjE5JZI/CPwP59+7VmzVr7dsK0g0os7g8EEEAAAQQQQCBqBOiRFTVLzUQRQMCJwJlUUGU1bnYqEmzPrJSYrqYBvJPYOQeBqBMIBEbGxgYG0BPLmyvv9n7qzVkSFQIIIIAAAgiEWoAeWaEW53oIIOBpgTOtoArmpOLi4goGArG1UxS4JCYQky+Y12JsBLwuEIgJHIhVzLqYmJQ5iYmJe70ebzTH58X9NJrXg7kjgAACCCAQKQL0yIqUlWQeCCDgikB2KqgyCoCKBFeWhUEQQMCHAm7vpz4kIGQEEEAAAQQQCIIAPbKCgMqQCCCAQKoAFQncCwgggAACCCCAAAIIIICAewL0yHLPkpEQQCACBNyuoKIiIQJuCqaAAAJnJOD2fnpGQfAhBBBAAAEEEIg4AXpkRdySMiEEEMiOABVU2dHjswgggMD/C7CfcjcggAACCCCAQDAE6JEVDFXGRAAB3wq4XUFFRYJvbwUCRwCBbAq4vZ9mMxw+jgACCCCAAAIRIkCPrAhZSKaBAALeFKAiwZvrQlQIIIAAAggggAACCCDgTwF6ZPlz3YgaAQSCJOB2BRUVCUFaKIZFAAHPC7i9n3p+wgSIAAIIIIAAAiERoEdWSJi5CAII+EWACiq/rBRxIoCA1wXYT72+QsSHAAIIIICAPwXokeXPdSNqBBAIkoDbFVRUJARpoRgWAQQ8L+D2fur5CRMgAggggAACCIREgB5ZIWHmIgggEK0CVCRE68ozbwQQQAABBBBAAAEEEAiGAD2ygqHKmAgg4FsBtyuoqEjw7a1A4AggkE0Bt/fTbIbDxxFAAAEEEEAgQgTokRUhC8k0EEDAHQEqqNxxZBQEEECA/ZR7AAEEEEAAAQSCIUCPrGCoMiYCCPhWwO0KKioSfHsrEDgCCGRTwO39NJvh8HEEEEAAAQQQiBABemRFyEIyDQQQ8KYAFQneXBeiQgABBBBAAAEEEEAAAX8K0CPLn+tG1AggECQBtyuoqEgI0kIxLAIIeF7A7f3U8xMmQAQQQAABBBAIiQA9skLCzEUQQMAvAlRQ+WWliBMBBLwuwH7q9RUiPgQQQAABBPwpQI8sf64bUSOAQJAE3K6goiIhSAvFsAgg4HkBt/dTz0+YABFAAAEEEEAgJAL0yAoJMxdBAIFoFaAiIVpXnnkjgAACCCCAAAIIIIBAMATokRUMVcZEAAHfCrhdQUVFgm9vBQJHAIFsCri9n2YzHD6OAAIIIIAAAhEiQI+sCFlIpoEAAu4IUEHljiOjIIAAAuyn3AMIIIAAAgggEAwBemQFQ5UxEUDAtwJuV1BRkeDbW4HAEUAgmwJu76fZDIePI4AAAggggECECNAjK0IWkmkggIA3BahI8Oa6EBUCCCCAAAIIIIAAAgj4U4AeWf5cN6JGAIEgCbhdQUVFQpAWimERQMDzAm7vp56fMAEigAACCCCAQEgE6JEVEmYuggACfhGggsovK0WcCCDgdQH2U6+vEPEhgAACCCDgTwF6ZPlz3YgaAQSCJOB2BRUVCUFaKIZFAAHPC7i9n3p+wgSIAAIIIIAAAiERoEdWSJi5CAIIRKsAFQnRuvLMGwEEEEAAAQQQQAABBIIhQI+sYKgyJgII+FbA7QoqKhJ8eysQOAIIZFPA7f00m+HwcQQQQAABBBCIEAF6ZEXIQjINBBBwR4AKKnccGQUBBBBgP+UeQAABBBBAAIFgCNAjKxiqjI
```

Comment:
In this diagram, `Enterprise Search` should be `Vertex AI Search`.

Reply:
I didn't make the Diagram... So that will need to be changed by the original author

Reply:
No worries, `Enterprise Search` is clear enough for now.